packages/next/src/build/webpack-build.ts

```diff
@@ -62,6 +63,12 @@ async function webpackBuildImpl(): Promise<{
   const dir = NextBuildContext.dir!
   const config = NextBuildContext.config!
 
+  const binding = await loadBindings()
+
+  if (!binding?.isWasm && typeof binding?.turbo?.nextBuild === 'function') {
+    await binding.turbo.nextBuild(NextBuildContext)
+  }
```

Comment:
This part should go into a `turbopack-build.ts`. We should switch the implementation [here](https://github.com/vercel/next.js/blob/e7e6f000fde3e927f04dfeda57e12924164ddaf8/packages/next/src/build/index.ts#L1012-L1013) or anywhere above this, using a `--turbo` flag passed from the CLI.